repos/spack_repo/builtin/packages/cp2k/package.py

```diff
@@ -225,8 +229,8 @@ class Cp2k(MakefilePackage, CMakePackage, CudaPackage, ROCmPackage):
     depends_on("greenx", when="+greenx")
     depends_on("hdf5+hl+fortran", when="+hdf5")
     depends_on("trexio", when="+trexio")
-    depends_on("libvdwxc", when="+vdwxc")
```

Comment:
The variant is still needed to explicitly set `CP2K_USE_LIBVDWXC` but the dependency is taken care of by SIRIUS, is this correct?

Reply:
true. Thanks for fixing this.